identifier/src/containers/Login/Login.jsx

```diff
@@ -83,6 +83,10 @@ function Login(props) {
       history.replace(`/chooseaccount${history.location.search}${history.location.hash}`);
       return;
     }
+
+    if(query && query.login_hint){
+      dispatch(updateInput("username", query.login_hint))
```

Comment:
This value comes from the URL and is untrusted. It must be sanitized so that it is safe.

Reply:
Updated with sanitization over login_hint.

Reply:
I don't see how this replace would do anything useful. Please remove it - not sure what am correct validation would be. Potential exploit vectors need to be investigated and in the commit message explained why the code is safe.

Reply:
Removed the local santizier and updated the sanitizer with dom purify which is a widely used plugin.

https://www.npmjs.com/package/dompurify